pkg/keystore/keystore.go

```diff
@@ -46,6 +46,9 @@ type Keystore struct {
 
 // New returns a new Keystore based on config.
 func New(config *Config) *Keystore {
```

Comment:
I am going to propose we just have New() and NewFromConfig(config *Config) since it seems like the former will be by far more common?